# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/css/css-text/white-space/text-wrap-balance-line-clamp-001.html.ini
# Commit: 16a3de29e4e6
# Full Hash: 16a3de29e4e69175f5a7c914ffa9de63b8683f2a
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2023-09-28 21:51:27
# Regressor Bug: 1731541
# File Overlap Count: 1
# Description:
#   Bug 1731541 - When line-clamp is in effect, make text-wrap:balance consider only the lines up to the clamp limit. r=emilio
#   
#   This corresponds to how Chrome behaves, and passes the test they included in WPT.
#   
#   It's unclear to me whether this behavior actually follows from the current spec
# ==============================================================================

diff -r 225642ff1de8 -r 16a3de29e4e6 testing/web-platform/meta/css/css-text/white-space/text-wrap-balance-line-clamp-001.html.ini
--- a/testing/web-platform/meta/css/css-text/white-space/text-wrap-balance-line-clamp-001.html.ini	Thu Sep 28 17:28:19 2023 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,2 +0,0 @@
-[text-wrap-balance-line-clamp-001.html]
-  expected: FAIL
